### PR TITLE
nfd-master: check that namespace informer cache sync succeeded

### PR DESCRIPTION
**Pull request by ozhuraki**
Mitigate the namespace cache sync reported in issue #1936

### CODE REVIEW

File: pkg/nfd-master/nfd-api-controller_test.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ func TestIsNamespaceSelected(t *testing.T) {
 
 	for _, tc := range testcases {
 		labelMap, _ := metav1.LabelSelectorAsSelector(tc.nodeFeatureNamespaceSelector)
-		c.namespaceLister = newNamespaceLister(fakeCli, labelMap)
+		c.namespaceLister, _ = newNamespaceLister(fakeCli, labelMap)
```
Comment (marquiz) [inline pkg/nfd-master/nfd-api-controller_test.go:107]: nit: safest to add an assert to the error, too

**Comment by ozhuraki** [comment]
@marquiz

Thanks, updated. Please take a look

**Review by marquiz — APPROVED** [review]
Thanks @ozhuraki, looks good to me
/assign @ArangoGutierrez

**Comment by marquiz** [comment]
ping @ArangoGutierrez @fmuyassarov

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Pull request merged**